htt_viz/NodeBitmask owner
uint8 active
uint8 done
float32 activation_level
float32 activation_potential
uint8 peer_active
uint8 peer_done
NodeBitmask highest
float32 highest_potential
int32 parent_type
float32 suitability
bool collision
bool robotPlacing
bool humanPlacing
bool peerUndone
string simstate_obj_name
geometry_msgs/Pose simstate_obj_pose
geometry_msgs/Pose simstate_robot_pose
geometry_msgs/Pose simstate_robot_goal


